setup.py

```diff
@@ -0,0 +1,22 @@
+from setuptools import setup
+
+
+requires = open("requirements.txt").read().split()
```

Comment:
this causes error while installation of module `no requirements.txt file found`.